db/migrate/20210722111142_remove_rhsm_columns.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveRhsmColumns < ActiveRecord::Migration[6.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    change_table "miq_databases" do |t|
+      t.remove "registration_type"
+      t.remove "registration_organization"
+      t.remove "registration_server"
+      t.remove "registration_http_proxy_server"
+    end
+
+    Authentication.where(:authtype => %w[registration registration_http_proxy])
+                  .destroy_all
+
+    change_table "authentications" do |t|
+      t.remove "rhsm_sku"
+      t.remove "rhsm_pool_id"
+      t.remove "rhsm_server"
+      #t.remove "manager_ref"
```

Comment:
This column should stay because it is for provider private key pairs. cc @agrare

Reply:
:+1: when we collect ssh keys from a provider we use this to store the provider native unique id, definitely still used